azurelinuxagent/common/protocol/hostplugin.py

```diff
@@ -480,8 +480,9 @@ def format_message(msg):
                 logger.info(message)
                 add_event(op=WALAEventOperation.HostPlugin, message=message, is_success=True)
 
-            # Don't support HostGAPlugin versions older than 115
-            if vm_settings.host_ga_plugin_version < FlexibleVersion("1.0.8.115"):
+            # Don't support HostGAPlugin versions older than 123
```

Comment:
123 contains all the fixes we need on the HostGAPlugin before releasing to Prod. Many VMs in the canary region are still on 117 so using that for now